### ISSUE TITLE
CTA inverse card in CardSectionSimple component: heading color does not change on hover

### ISSUE THREAD

**Issue by Daynssjal** [carbon-design-system/carbon-for-ibm-dotcom#5663]
Hi, not sure if this is a bug or it works as designed, but it is not very good for accessibility.
The CTA inverse card (last card) in CardSectionSimple component does not change color of the heading text on hover.
It stays light, although the background of the card changes to light, and so the heading is not properly visible.

1. Go to https://ibmdotcom-react.mybluemix.net/?path=/story/components-cardsectionsimple--with-cta
2. Hover over the last dark card.
3. Check how the heading text is very poorly visible as the card becomes light

What did you expect to happen? What happened instead? What would you like to see changed?
The heading text will get a darker color on hover, to be more visible

What browser are you working in?
Chrome, FF, Safari, Edge

What version of Carbon for IBM.com are you using?
1.17.0

What offering/product do you work on? Any pressing ship or release dates we should be aware of?
Knowledge Center

- Screenshots or code
- Notes
<img width="747" alt="inverse-card-on-hover" src="https://user-images.githubusercontent.com/33804471/112949933-03a1d400-913a-11eb-8e01-9861e859c21c.PNG">
<img width="777" alt="inverse-card" src="https://user-images.githubusercontent.com/33804471/112949938-043a6a80-913a-11eb-836b-70b854c44ff7.PNG">

**Comment by DragosRistici** [comment]
Both React and Web Components Storybooks, Card section simple – with CTA, on all Carbon themes, on hover over the CTA, the expected container background is not present.

<img width="1569" alt="Screenshot 2021-03-30 at 11 58 51" src="https://user-images.githubusercontent.com/48327647/112962768-6782c780-914f-11eb-88f9-9f6f7f25b8a8.png">
